js/gated-content/src/views/LiveStreamPage.vue

```diff
@@ -123,12 +123,17 @@ export default {
     descriptionProcessed() {
       return this.description ? this.description.processed : '';
     },
+    descriptionTruncated() {
+      return `${this.descriptionProcessed.length > 1000}`
```

Comment:
Please, use mixing here.
Since we are using equal code at 2 places, lets store it at our mixin for events:
All that you need is to add your descriptionTruncated code to this file: 
https://github.com/fivejars/openy_gated_content/blob/master/js/gated-content/src/mixins/EventMixin.js#L22
And delete it here.
Code below in `description` will automatically use your computed var code from mixing.